staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -24,6 +24,7 @@ import (
 	goflag "flag"
 	"fmt"
 	"io"
+	golog "log"
```

Comment:
This is not `go-kit/log`, you should use the already imported `klog`

Reply:
nit: I think you can drop the alias here, it should not conflict with `go-kit`.